extension/registry/holixon/holixon-axon/src/main/avro/holixon.registry.AvroSchemaRegistered-v1.avsc

```diff
@@ -0,0 +1,49 @@
+{
+  "type": "record",
+  "namespace": "holixon.registry.event",
+  "name": "AvroSchemaRegisteredEvent",
+  "doc": "A new schema has been registered.",
+  "revision": "1",
+  "fields": [
+    {
+      "name": "schemaId",
+      "doc": "Schema id",
+      "type": {
+        "type": "string",
+        "avro.java.string": "String"
+      }
+    },
+    {
+      "name": "namespace",
+      "doc": "Namespace of the schema.",
+      "type": {
+        "type": "string",
+        "avro.java.string": "String"
+      }
+    },
+    {
+      "name": "name",
+      "doc": "Name of the schema.",
+      "type": {
+        "type": "string",
+        "avro.java.string": "String"
+      }
+    },
+    {
+      "name": "revision",
+      "doc": "Optional revision of the schema.",
```

Comment:
Since this is optional, we need a union type here 

{"name": "revision", "type": ["null", "string"]} // optional next element

Reply:
Good point... A unit test would detect this failure hopefully...